website/index.html

```diff
@@ -9,13 +9,12 @@
       <div class="container">
           <div class="row">
               <div class="col-md-12 text-center">
-                  <h2>Make Your Company Data Driven</h2>
-                  <h3>Connect to any data source, easily visualize and share your data</h3>
+                  <font color="F2F9FC"><h2>Make Your Company Data Driven</h2>
+                  <h3>Connect to any data source, easily visualize and share your data</h3></font>
                   <p id="buttons">
-                      <a href="https://demo.redash.io/" data-track data-track-location="hero" data-track-event="Clicked Demo"  class="btn btn-primary btn-lg btn-redash">Demo</a>
-                      <a href="/#get-started" data-track data-track-location="hero" data-track-event="Clicked Get Started" class="btn btn-primary btn-lg btn-redash">Get Started</a>
-                  </p>
-
+                      <a href="/#get-started" data-track data-track-location="hero" data-track-event="Clicked Get Started" class="btn btn-primary btn-lg btn-redash">Get Started</a></p>
+                      <a href="https://demo.redash.io/" data-track data-track-location="hero" data-track-event="Clicked See Plans"><font color="FFFFFF">See Plans</font></a>
+<p></p>
```

Comment:
Let's revert this as well - before dropping the demo link I would like us to think of metric to track to see if it had impact (positive or negative).